addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessage.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+/**
+ * RFXCOM data class for control message.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComInterfaceControlMessage extends RFXComBaseMessage {
+
+    private byte[] data = new byte[14];
+
+
+    public RFXComInterfaceControlMessage(RFXComInterfaceMessage.TransceiverType transceiverType, RFXComBridgeConfiguration configuration) {
+        data[0] = 0x0D;
+        data[1] = RFXComBaseMessage.PacketType.INTERFACE_CONTROL.toByte();
+        data[2] = 0;
+        data[3] = 2;
+        data[4] = RFXComInterfaceMessage.Commands.SET_MODE.toByte();
+        data[5] = transceiverType.toByte();
+        data[6] = 0;
```

Comment:
I think 0x1C is the maximum value. The _correct_ value is the smallest value that will allow your furthest receivers to operate reliably without spamming your neighbours unnecessarily. Currently anyone that needs higher will have to continue using a hex string or set a saved config with rfxmgr. There really needs to be a transmit power config option but that might be better as a separate issue/PR since it's a feature add.